rule_packages/c/SideEffects4.json

```diff
@@ -0,0 +1,23 @@
+{
+  "CERT-C": {
+    "PRE31-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Macro arguments can be expanded multiple times which can cause side-effects to be evaluated multiple times.",
+          "kind": "problem",
+          "name": "Avoid side effects in arguments to unsafe macros",
+          "precision": "low",
```

Comment:
maybe add implementation note for the details of this precision, ie as discussed in meeting that we do not look for the no use of a parameter case

Reply:
Done.